### ISSUE TITLE
[Dinkur CLI] Ask user on AFK-notification

### ISSUE THREAD

**Issue by jilleJr** [dinkur/dinkur#16]
Depends on: #15 

The Dinkur CLI should ask the user before performing any task if there is a pending notification for how to deal with it. Ex:

```console
$ dinkur in Doing some other work
Note: You were away since 13:31 - now… (0:25:31)
while having an active task "My previously active task" 13:01 - now… (0:56:40)
How do you want to save this away time?

[1] Leave the active task as-is. Make no modifications.
[2] Discard the time I was away, changing the active task to 13:01 - 13:31 (0:31:09).
[3] Save the time away as a new task 13:31 - now… (0:25:31) (naming it in a later step).
[4] Split the time away into multiple tasks (interactively).

Enter option [1-4]: 1
Leaving active task as-is: "My previously active task" 13:01 - now… (0:56:40)
Continuing with command...

Stopped task: "My previously active task" 13:01 - 13:57 (0:56:40)
Started task: "Doing some other work" 13:57 - now…
```

If the Dinkur CLI isn't executed in an interactive session, then just log something like:

```console
Note: You were away since 13:31 - now… (0:25:31)
while having an active task "My previously active task" 13:01 - now… (0:56:40)
Warning: Session is not interactive, so assuming "Leave the active task as-is. Make no modifications."
Leaving active task as-is: "My previously active task" 13:01 - now… (0:56:40)
Continuing with command...

Stopped task: "My previously active task" 13:01 - 13:57 (0:56:40)
Started task: "Doing some other work" 13:57 - now…
```
